content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md

```diff
@@ -18,32 +18,59 @@ If the network uses a directory service, import the existing account information
-To see built-in groups, click the **Show Built-In Groups** toggle. The toggle turns blue, and all built-in groups display. Click the **Show Built-In Groups** toggle again to show only non-built-in groups on the system.
+To see built-in groups, click the **Show Built-In Groups** toggle. The toggle turns blue and all built-in groups display. Click the **Show Built-In Groups** toggle again to show only non-built-in groups on the system.
 
 ## Adding a New Group
 
 To create a group, go to **Credentials > Local Groups** and click **Add**.
 
-![AddGroupGIDConfigSCALE](/images/SCALE/22.12/AddGroupGIDConfigSCALE.png "Add Group") 
+{{< trueimage src="/images/SCALE/23.10/AddGroupGIDConfigSCALE.png" alt="Add Group Screen" id="2: Add Group Screen" >}}
 
-Enter a unique number for the group ID in **GID** that TrueNAS uses to identify a Unix group. Enter a number above 1000 for a group with user accounts or for a system service enter the default port number for the service as the GID.  Enter a name for the group. The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal (=). You can only use the dollar sign ($) as the last character in a user name.
+Enter a unique number for the group ID in **GID** that TrueNAS uses to identify a Unix group.
+Enter a number above 3000 for a group with user accounts or enter the default port number as the GID for a system service.
 
-**Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allows the group members to act as the system administrator using the [sudo](https://www.sudo.ws/) command. Leave it disabled for better security.
+Enter a name for the group.
+The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal (=).
+You can only use the dollar sign ($) as the last character in a group name.
+
+**Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allow members of the group to act as root using the [sudo](https://www.sudo.ws/) command.
+To allow sudo commands, use **Allowed sudo commands** or **Allowed sudo commands with no password** to list the executable path(s) to specific sudo commands allowed for group members.
+<file>/usr/bin/</file> is the default location for command executables.
+Alternately, click the checkbox for **Allow all sudo commands** or **Allow all sudo commands with no password**.
```

Comment:
style guide says to not say checkbox unless there is no other way, also when it is a checkbox use select, for example
```suggestion
Alternately, Select **Allow all sudo commands** or **Allow all sudo commands with no password**.
```